### ISSUE TITLE
[ERROR] 502: Bad Gateway

### ISSUE THREAD

**Issue by agamsol** [NoNameoN-A/TikTok-Follow-Heart-Views-Bot#114]
Cannon access https://vipto.de/ at all
a list of things i tried:
- open https://vipto.de/ with Surfshark VPN on berlin
- open https://vipto.de/ in normal browser 
- open https://vipto.de/ in VM
- open https://vipto.de/ in iPhone
- Change the agent to safari in mac [using this extension](https://chrome.google.com/webstore/detail/user-agent-switcher/aedikcfpfonanffanecfolneiaoakmlc?hl=en#:~:text=User%2DAgent%20Switcher%20is%20a,add%20your%20own%20user%2Dagents)
- change `driver.get("https://vipto.de/")` --> `driver.get(vidUrl)` in line 163 (when i verified nothing happened in the CMD window)

**To Reproduce**

basically accessing the website to see if it works from regular chrome

OR from bot:
1. type `py bot.py`' in cmd.exe
2. Click `1` or whatever other option that will open
3. the site wont load ![image](https://i.imgur.com/o1run2o.png)

**Expected behavior**
The site should load to complete the verification

**Info**
ChromeDriver: ChromeDriver 94.0.4606.61
Chrome Browser: Version 94.0.4606.61
OS: Windows 10
